api/controllers/linux/install/infra.go

```
@@ -67,7 +67,16 @@ func (c *InstallController) SetupInfra(ctx context.Context, ignoreHostPreflights
 			}
 		}()
 
-		if err := c.infraManager.Install(ctx, c.rc); err != nil {
+		configValues := make(map[string]string)
+		if c.store != nil && c.store.AppConfigStore() != nil {
+			if values, err := c.store.AppConfigStore().GetConfigValues(); err != nil {
+				c.logger.WithError(err).Warn("reading config values from store")
```

this should return an error, not just log it